<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="Canvas" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>0</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="centerXField">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="centerYField">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>840</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Центр фигуры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="axisXField">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="axisYField">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Полуоси эллипса</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>a</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
      </font>
     </property>
     <property name="text">
      <string>b</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="scaleCenterLabel_6">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Алгоритм</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Каноническое уравнение</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Параметрическое уравнение</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Алгоритм Брезенхема</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Алгоритм  средней точки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Функция стандартной библиотеки</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="figureDrawButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начертить фигуру</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="scaleCenterLabel_7">
     <property name="font">
      <font>
       <family>Hack Nerd Font Mono</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Цвет</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="colorComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>401</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Красный</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Оранжевый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Желтый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Зеленый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Голубой</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Синий</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Фиолетовый</string>
     </property>
    </item>
   </widget>
   <widget class="QSplitter" name="splitter_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>231</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="rangeDrawButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начертить спектр</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Количество фигур</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Шаг</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountField">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="stepField">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Радиус окружности</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusField">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>102</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_author">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Об авторе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prog">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleCenterLabel_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Hack Nerd Font Mono</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фигура</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="figureTypeComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Окружность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
